src/client/mod.rs

```diff
@@ -827,7 +827,7 @@ impl KafkaClient {
             for tp in resp.topic_partitions {
                 let e = res.entry(tp.topic).or_insert(vec![]);
                 for p in tp.partitions {
-                    e.push(p.into_offset());
+                    e.push(try!(p.into_offset()));
```

Comment:
Yeah, this was part of our earlier discussion. I could certainly add another error type, provided you don't mind the extra time it will take to get this PR merged. I personally lean toward just doing it right the first time, but I'm also ok with just adding the debug line and adding the error type in a future PR. I leave it up to you :)